code/migration/src/main/scala/com/namely/chiefofstate/migration/versions/v1/V1.scala

```diff
@@ -74,17 +62,15 @@ case class V1(
    */
   override def upgrade(): DBIO[Unit] = {
     // get the correct table name
-    val table: String = transformTableName()
+    val oldTable: String = transformTableName()
     log.info(s"finalizing ChiefOfState migration: #$versionNumber")
     DBIO.seq(
       // dropping the old table in the new projection connection
-      sqlu"""DROP TABLE IF EXISTS #$table""",
+      sqlu"""DROP TABLE IF EXISTS #$oldTable CASCADE""",
       // renaming the temporary table in the new projection connection
-      sqlu"""ALTER TABLE IF EXISTS #$tempTable RENAME TO #$table""",
-      // dropping the temporary table index
-      sqlu"""DROP INDEX IF EXISTS #${tempTable}_index""",
+      sqlu"""ALTER TABLE #$tempTable RENAME TO #$offsetTableName""",
       // creating the required index on the renamed table
-      sqlu"""CREATE INDEX IF NOT EXISTS projection_name_index ON #$table (projection_name)"""
+      sqlu"""CREATE INDEX #${offsetTableName}_projection_name_index ON #$offsetTableName (projection_name)"""
```

Comment:
I will suggest to maintain the index name as set by lightbend to avoid unforseen behaviour. Yeah it is just a name. However we don't know how it is used in the projection code.